app/scripts/metamask-controller.js

```diff
@@ -5158,9 +5271,17 @@ export default class MetamaskController extends EventEmitter {
   }
 
   _notifyChainChange() {
```

Comment:
looks like we're passing `newState` to this method on line 4919 but not actually using it here?